_docs/datasources/google sheets.md

```diff
@@ -22,13 +23,54 @@ In this section you will learn:
   <li><a href="#column-types">Column types</a></li>
   <li><a href="#destination-table-name">Destination table name</a></li>
   <li><a href="#saving-disconnecting-disabling">Saving, Disconnecting, Disabling data source</a></li>
+  <li><a href="#example">Google Sheets data source setup example</a></li>
 </ul>
 
 ### <a name="importing-data"></a>Importing data from Google Sheets
 ------
 
 After enabling this data source, data from Google Sheets will be uploaded to BigQuery once every 24 hours.
 
+### <a name="data-requirements"></a>⚠️ Data requirements:
+------
+- You can only import data formatted as simple table. That means that chosen range should have at least two rows:
+one header row and one or more data rows.
+
+- First non-empty row in the range would be considered header.
+
+- All subsequent rows will be considered data rows. Table end is detected by the first empty row after data rows.
```

Comment:
> Table end is detected by the first empty row after data rows.

I understand what you mean, but this line confusing me.

F.e. I have table:
Header 1 | Header 2
 --- | --- | 
cell 1 |  cell 2| 
&#65279; | &#65279; |
cell 3 | cell 4| 

When I read the doc, I can't understand where my data rows? Just 1 row (because 2nd row has empty values) or 3.